src/WebJobs.Script/Config/ConfigurationSectionNames.cs

```diff
@@ -13,5 +13,6 @@ public static class ConfigurationSectionNames
         public const string HostIdPath = WebHost + ":hostid";
         public const string ExtensionBundle = "extensionBundle";
         public const string ManagedDependency = "managedDependency";
+        public const string Hsts = "hsts";
```

Comment:
Should this live under "http"?

Reply:
Yeah, i think you are right this should live under http.